.gitignore

```diff
@@ -79,7 +79,10 @@ documentation/bolt_command_reference.md
 documentation/bolt_configuration_reference.md
 documentation/privilege_escalation.md
 documentation/bolt_defaults_reference.md
+documentation/bolt_pwsh_reference.md
 documentation/bolt_project_reference.md
 documentation/bolt_transports_reference.md
 tasks/
 plans/
+
+pwsh_module/pwsh_bolt.psm1
```

Comment:
Should this also include `pwsh_module/autogenerated.tests.ps1`?

Reply:
I was initially troubled by it not being present to troubleshoot a CI failure, but realized we are running `bundle exec rake pwsh:generate_module` to get the psm1 file anyway so having the autogenerated tests present doesn't make much sense.

The only concern I have left is that we aren't tracking changes to either the psm1 or the tests, so we won't have a diff of what changes from one release to another

Reply:
I think that's probably ok, since we can diff packages? It's a bit more cumbersome, but we're not committing the generated module anyway, so it doesn't make sense to commit the tests.